TFT/src/User/API/Language/Language.inc

```diff
@@ -17,6 +17,7 @@ X_WORD (LANGUAGE)
 X_WORD (ACK_NOTIFICATION)
 X_WORD (FILES_SORT_BY)
 X_WORD (FILES_LIST_MODE)
+X_WORD (FILES_FULLNAME_VIEW)
```

Comment:
It would be better to change it `FILES_NAME_EXTENSIONS`

Reply:
changed to FILENAME_EXTENSION